utils/rtc_clock_menu.c

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (C) 2022 PC Engines GmbH
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
+ */
+
+#include <libpayload.h>
+#include <curses.h>
+
+static void print_rtc_clock_menu(struct tm *t) {
```

Comment:
```suggestion
static void print_rtc_clock_menu(struct tm *t)
{
```
Brackets of functions should start in new line. Please fix for all other functions

Reply:
Fixed